test/performance/dataplane-probe/main.go

```diff
@@ -47,10 +48,18 @@ func main() {
 	ctx, cancel := context.WithTimeout(ctx, 6*time.Minute)
 	defer cancel()
 
+	tags := []string{"master"}
+	commitID, err := changeset.Get()
+	if err == nil {
+		tags = append(tags, commitID)
+	} else {
+		log.Printf("Error getting commit id: %v", err)
```

Comment:
Can we use `log.Fatalf(...)` here?

Reply:
We can simplify this a bit too:
```
tags := []string{"master"}
if commitID, err := changeset.Get(); err == nil {
  tags = append(tags, commitID)
} else {
  log.Fatalf(...)
}
```

Reply:
Fataling seems redundant here, I think

Reply:
Ahh right, it shouldn't be fatal here. My bad 😅